chapter12/sxg/ex_12_2.pl

```diff
@@ -0,0 +1,23 @@
+#!perl
+use warnings;
+use strict;
+
+die "No file names found!\n" unless @ARGV;
+
+my $old_name = shift @ARGV;
+my $old_age = -M $old_name;
+$old_age = 0 if (!defined $old_age) ;

```

Comment:
Regarding the age.. I always believed the age of a thing is the age of last modification. :) If i buy an old car and get a new part the car is new :d